src/jwplayer/types.d.ts

```diff
@@ -439,6 +450,7 @@ export interface JwPlayerWrapperProps extends JwPlayerContainerId {
 	className?: string;
 	stopAutoAdvanceOnExitViewport?: boolean;
 	shouldLoadSponsoredContentList?: boolean;
+	vastUrl?: boolean | string;
```

Comment:
Why should this property be a boolean?

Reply:
Good question. It should be string 🤔 I'll take a look 👍